sdk/metric/metricdata/data.go

```diff
@@ -211,6 +212,19 @@ type Extrema[N int64 | float64] struct {
 	valid bool
 }
 
+// MarshalText converts the Extrema value to text.
+func (e Extrema[N]) MarshalText() ([]byte, error) {
+	if !e.valid {
+		return []byte("0"), nil
```

Comment:
@MrAlias is it a correct reprentation for an invalid extrema? Shouldn't we return nil, nil?

Reply:
@MrAlias nil will output empty bytes, which will break MarshalJSON. Can we settle on `return json.Marshal(nil)` for `MarshalText`? 
Text representation will then be equal to `null`, not to just to no bytes.